tests/testthat/helper-demo.R

```diff
@@ -1,11 +1,23 @@
 #### task-plan demo ####
 
 setup_tasks_demo <- function() {
+  random_string <- function() {
+    random_letters <- sample(LETTERS, 7, TRUE)
+    random_numbers <- sample(0:9, 5, TRUE)
+    paste(sample(c(random_letters, random_numbers), 10, FALSE), collapse='')
+  }
+  
+  find_srcdir <- function() {
+    srcdir <- system.file('inst/extdata/tasks_demo', package='scipiper') # for devtools::test()
+    if(srcdir == '') srcdir <- system.file('extdata/tasks_demo', package='scipiper') # for local tests (and R CMD check?)
+    return(srcdir)
+  }
+  
   dirinfo <- list(
     wd = getwd(),
     #srcdir = 'inst/extdata/tasks_demo', # for use while developing tests
-    srcdir = system.file('inst/extdata/tasks_demo', package='scipiper'),
-    tmpdir = file.path(tempdir(), format(Sys.time(), '%y%m%d_%H%M%S')))
+    srcdir = find_srcdir(),
+    tmpdir = file.path(tempdir(), random_string()))
```

Comment:
can you just use `tempfile` here?

Reply:
Hmm. I was thinking I couldn't because `tempfile` sounds like it's proposing a file name, whereas I want a directory name...but yes, you're right, tempfile should work because it's just an extensionless name. Thanks, will change in next PR